pkg/debian/build.sh

```diff
@@ -68,10 +68,28 @@ fakeroot dpkg-deb --build "${SERVERROOT}" "${DISTROOT}/${APP_NAME}-server_${APP_
 echo "Creating the desktop package..."
 mkdir "${DESKTOPROOT}/DEBIAN"
 
-cat << EOF > "${DESKTOPROOT}/DEBIAN/conffiles"
+# Ubuntu 24 requires apparmor profile to work.
+UBUNTU_CODENAME=$(grep "^UBUNTU_CODENAME=" /etc/os-release | awk -F "=" '{ print $2 }')
+if [ "${UBUNTU_CODENAME}" == 'noble' ]; then
+  cat << EOF > "${DESKTOPROOT}/DEBIAN/conffiles"
 /etc/apparmor.d/pgadmin4
 EOF
 
+  mkdir -p "${DESKTOPROOT}/etc/apparmor.d"
+  cp "${SOURCEDIR}/pkg/debian/pgadmin4-aa-profile" "${DESKTOPROOT}/etc/apparmor.d/pgadmin4"
+
+  cat << EOF > "${DESKTOPROOT}/DEBIAN/postinst"
+#!/bin/sh
+
+STATUS="$(systemctl is-active apparmor.service)"
+if [ "${STATUS}" == "active" ]; then
+    echo "Restarting apparmor.service to load pgAdmin profile..."
+    systemctl restart apparmor.service
```

Comment:
Do we need to restart it, or can we just reload it?

Reply:
OK I found a better way. This will only load pgadmin4 
```
apparmor_parser -r /etc/apparmor.d/pgadmin4
```